Frameworks/UIKit/UIWindow.mm

```diff
@@ -311,6 +255,21 @@ - (void)setRootViewController:(UIViewController*)controller {
     }
 }
 
+/**
+ @Status Interoperable
+*/
+- (BOOL)sizeUIWindowToFit {
+    return _sizeUIWindowToFit;
```

Comment:
Nice catch.  Indeed; it looks like an MS-extension that leaked into the public interface: [https://github.com/Microsoft/WinObjC/wiki/UI-Guidance](https://github.com/Microsoft/WinObjC/wiki/UI-Guidance)

UIApplication.mm uses it in some window/app sizing logic. I'll remove the annotations and add a comment to the header.

---
In reply to: [111611637](https://github.com/Microsoft/WinObjC/pull/2445#discussion_r111611637) [](ancestors = 111611637)